doc/bipart.xml

```diff
@@ -135,6 +135,48 @@ gap> NrRightBlocks(x);
   </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="DomainOfBipartition">
+  <ManSection>
+    <Attr Name = "DomainOfBipartition" Arg = "x"/>
+    <Returns>A list of positive integers.</Returns>
+    <Description>
+      If <A>x</A> is a bipartition, then <C>DomainOfBipartition</C> returns the
+      domain of <A>x</A>. The domain of <A>x</A> consists of those numbers
+      <C>i</C> in <C>[1 .. n]</C> such that <C>i</C> is contained in a
+      transverse block of <A>x</A>, where <C>n</C> is the degree of <A>x</A>
+      (see <Ref Attr="DegreeOfBipartition"/>).
+      <Example><![CDATA[
+gap> x := Bipartition([
+>  [1, 2] , [3, 4, 5, -5], [6, -6], [-1, -2, -3], [-4]]);
+<bipartition: [ 1, 2 ], [ 3, 4, 5, -5 ], [ 6, -6 ], [ -1, -2, -3 ], 
+ [ -4 ]>
+gap> DomainOfBipartition(x);
+[ 3, 4, 5, 6 ]]]></Example>
+    </Description>
+  </ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="CodomainOfBipartition">
+  <ManSection>
+    <Attr Name = "CodomainOfBipartition" Arg = "x"/>
+    <Returns>A list of positive integers.</Returns>
+    <Description>
+      If <A>x</A> is a bipartition, then <C>CodomainOfBipartition</C> returns
+      the codomain of <A>x</A>. The codomain of <A>x</A> consists of those
+      numbers <C>i</C> in <C>[1 .. n]</C> such that <C>-i</C> is contained in
+      a transverse block of <A>x</A>, where <C>n</C> is the degree of <A>x</A>
+      (see <Ref Attr="DegreeOfBipartition"/>).
+      <Example><![CDATA[
+gap> x := Bipartition([
+>  [1, 2] , [3, 4, 5, -5], [6, -6], [-1, -2, -3], [-4]]);
+<bipartition: [ 1, 2 ], [ 3, 4, 5, -5 ], [ 6, -6 ], [ -1, -2, -3 ], 
+ [ -4 ]>
+gap> CodomainOfBipartition(x);
+[ 5, 6 ]]]></Example>
+    </Description>
```

Comment:
Why are you returning positive numbers here, rather than negative? Is this so that `CodomainOfBipartition(x) = DomainOfBipartition(Star(x))`??